Automattic/woocommerce-payments
# Add e2e tests for the currency switcher widget

Closes #7349.

#### Changes proposed in this Pull Request

Add e2e tests for the currency switcher widget covering the multi-currency functionality.









#### Testing instructions





1. Set up a clean e2e local environment: `npm run test:e2e-reset` then `npm run test:e2e-setup`.
2. Run `npm run test:e2e -- admin-multi-currency multi-currency-widget`.
3. All tests should pass.
4. Confirm that all checks for e2e tests pass in this PR. Looks like the e2e check `WC - latest | wcpay - shopper` is consistently failing in other PRs too, you can see in the run that the failures are not related to this PR.

-------------------


**Post merge**
